Sources/Subprocess/IO/Output.swift

```diff
@@ -46,7 +46,7 @@ public protocol OutputProtocol: Sendable, ~Copyable {
 #endif
 extension OutputProtocol {
     /// The max amount of data to collect for this output.
-    public var maxSize: Int { 128 * 1024 }
+    public var maxSize: Int { .max }
```

Comment:
@iCharlesHu Why are we fixing a hang with increasing the buffer size to infinity? That's not quite the right fix I think.

if you're not streaming and you blow through the max size you should get an error, not a hang. That's also what AsyncProcess does.

in fact, you will now see a hang (followed by an OOM kill) if you run for example `cat /dev/zero`.

Reply:
Safe and unexpected is better than unsafe an unexpected IMHO. So a limit is IMHO a must.